share/server/util.js

```diff
@@ -116,7 +116,14 @@ var Couch = {
         "Expression does not eval to a function. (" + source.toString() + ")"]);
     };
   },
-  recursivelySeal : deepFreeze
+  recursivelySeal: function (obj) {
+    seal(obj);
+    for (var propname in obj) {
```

Comment:
What does this for-loop do?

Reply:
`propname` is the index, if the element type is `object`, this `function(obj) { seal(obj); ... }` will be called again with the new argument `obj = obj[propname]`.